includes/wcpdf-functions.php

```diff
@@ -721,3 +721,190 @@ function wpo_wcpdf_get_multilingual_languages(): array {
 	return apply_filters( 'wpo_wcpdf_multilingual_languages', $languages );
 }
 
+/**
+ * Get image mime type
+ *
+ * @param string $src
+ * @return string
+ */
+function wpo_wcpdf_get_image_mime_type( string $src ): string {
+	$mime_type = '';
+
+	if ( empty( $src ) ) {
+		return $mime_type;
+	}
+
+	// Check if 'getimagesize' function exists and try to get mime type for local files
+	if ( function_exists( 'getimagesize' ) ) {
+		$image_info = @getimagesize( $src );
```

Comment:
What do you think of having this approach at the bottom, maybe as the last chance? 

I thought maybe Fileinfo as the first approach is better, as suggested by PHP docs.
Also, the `getimagesize()` sends an HTTP request, which could be unnecessary. For low-speed internet connections, processing the mime type could take a long time (I waited almost 15s for a `176 KB` logo!).

Reply:
Was it a local or remote file?

Reply:
It was the logo in my local, but I was viewing the HTML version, so the path was like this:
`https://localhost/mohamad/wp-content/uploads/2024/04/DALL.png`

Reply:
What is the `$src` value passing [here](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/591f37c53990cd5ba9d1e209d66862fd622c39c1/includes/documents/abstract-wcpdf-order-document.php#L861)?

Reply:
```php
$attachment_src;
// https://localhost/mohamad/wp-content/uploads/2024/04/DALL.png


get_attached_file( $attachment_id ); 
// C:\laragon\www\mohamad/wp-content/uploads/2024/04/DALL.png

realpath(get_attached_file( $attachment_id )); 
// C:\laragon\www\mohamad\wp-content\uploads\2024\04\DALL.png

wp_normalize_path(realpath(get_attached_file( $attachment_id ))); 
// C:/laragon/www/mohamad/wp-content/uploads/2024/04/DALL.png

Reply:
Could you try my last commit?

Reply:
The logo is loading and also loading fast.

Reply:
FYI, for PDF output (which is displaying correctly), this is the `$src` I have by registering a log [here](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/826/files#diff-b4b45c08877ccc6f215e534426a81b3edd82ce371cfc3fce7420e62e279b3c1eR868):
`/laragon/www/mohamad/wp-content/uploads/2024/04/DALL.png`

Reply:
Yeah, I think it's not necessary to remove the drive letter. I don't see any issue after commenting that line.